src/main/scala/com/codacy/model/IssuesAnalysis.scala

```diff
@@ -0,0 +1,9 @@
+package com.codacy.model
+
+import java.nio.file.Path
+
+final case class ToolResults(tool: String, issues: IssuesAnalysis)
+
+final case class IssuesAnalysis(results: Set[FileResults])
```

Comment:
Is this case class needed?

Reply:
It is, we need it in order to serialize the json correctly